changelog.d/635.change.rst

```diff
@@ -0,0 +1,6 @@
+Added ``@attr.s(collect_by_mro=False`` argument that if set to ``True`` fixes the collection of attributes from base classes.
```

Comment:
dupe fragment?
```suggestion
Added ``@attr.s(collect_by_mro=False)`` argument that if set to ``True`` fixes the collection of attributes from base classes.
```

Reply:
Heh thanks for both, i fixed it myself when i saw your suggestion. :)